release/models/network-instance/openconfig-route-summary.yang

```diff
@@ -0,0 +1,96 @@
+module openconfig-route-summary {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/route-summary";
+
+  prefix "oc-rtsummary";
+
+  import openconfig-extensions { prefix "oc-ext"; }
+  import openconfig-yang-types { prefix "oc-yang-types"; }
+  import openconfig-policy-types { prefix "oc-pol-types"; }
+  import openconfig-network-instance { prefix "oc-ni"; }
+
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    www.openconfig.net";
+
+  description
+    "This module provides summary of route counts per route type for each network
+    instance.";
+
+  oc-ext:openconfig-version "1.0.0";
+
+  revision "2022-12-21" {
+    description
+      "Initial version.";
+    reference "1.0.0";
+  }
+
+  grouping route-counter {
+    leaf route-type {
+      type identityref {
+        base "oc-pol-types:INSTALL_PROTOCOL_TYPE";
+      }
+      description
+        "Route type that keys the route count list.";
+    }
+    leaf count {
+      type oc-yang-types:counter64;
+      description
+        "Count for the route type.";
+    }
+    description
```

Comment:
Please put this description at the top of the container for readability.

Reply:
Done and moved all descriptions to the top of the block in which they are defined.